website/app/components/dynamic-template.js

```diff
@@ -45,7 +45,7 @@ export default class DynamicTemplate extends Component {
         // if component couldn't be found the old way try importing it directly
         let module;
         try {
-          module = importSync(`/docs/${this.args.componentId}.js`);
+          module = importSync(`./docs/${this.args.componentId}.js`);
```

Comment:
(question) is this related to the other changes made in this PR?

Reply:
non, not at all, it was just something that keeps coming up if I `pnpm dedupe`, totally unrelated